src/Chapter1.hs

```diff
@@ -635,8 +646,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n 
+    | (abs n) < 10 = abs n
+    | otherwise = firstDigit (div (abs n) 10)
```

Comment:
Nice solution! I see that you use `abs n` several times. You can move it into a separate variable, to reduce some `()` and more code a bit cleaner :slightly_smiling_face: